vendor/folio_deployment/lib/folio_deployment/cli/commands/base.rb

```diff
@@ -0,0 +1,87 @@
+# coding: utf-8
+module FolioDeployment
+  module CLI
+    module Commands
+      class Base < Clamp::Command
+        attr_accessor :pastel, :prompt, :shell, :client, :okapi, :dnsimple_client
+
```

Comment:
i'm down with this as-is and then we can hit this in a future iteration, but this is really where we should be loading the configuration.